R/messages.R

```diff
@@ -153,6 +153,12 @@ messages$stopNoPathProvided <- function(dataCombinedName) {
   )
 }
 
+messages$stopWrongOutputPath <- function(dataCombinedName, scenarioName, path) {
+  paste0(
```

Comment:
Maybe we could use {glue} in the future for long strings like this.

Reply:
@Felixmil feel free to make a suggestion, I would be glad to learn about it and I think in the `messages` there are many use cases for this.